packages/presentation/lib/src/di/presentation_injector.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:domain/domain.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:presentation/src/game_bloc/game_bloc.dart';
+
+Future<void> initPresentationModule() async {
+  initMainPageModule();
+}
+
+void initMainPageModule() {
+  // for reviewers...
+  // i decided to use singleton to avoid pass a GameBloc instance
+  // to widget constructors, which lay lower on widget tree
+  // same for Global key
+  GetIt.I.registerLazySingleton<GameBloc>(
+    () => GameBloc(
+      GetIt.I.get<GenerateGuessNumberUseCase>(),
+      GetIt.I.get<MakeAttemptUseCase>(),
+    ),
+  );
```

Comment:
not the best option.

why? 
1) because if your app will have a lot of screens, bloc won't be destroyed after closing screen
2) when u will navigate to the screen again u should have new bloc instance (imho)